frontend/scripts/react-components/profiles/actor-summary.component.jsx

```diff
@@ -49,74 +138,14 @@ class ActorSummary extends React.PureComponent {
           <div className="small-12 columns">
             <TitleGroup titles={titles} on={onYearChange} />
           </div>
-          <div className="small-12 columns">
-            {typeof forest500 !== 'undefined' && (
-              <div className="stat-item">
-                <div className="legend">
-                  <Text transform="uppercase" variant="mono" as="span">
-                    Forest 500 score
-                  </Text>
-                  <span id="forest-500-tooltip">
-                    <HelpTooltip
-                      text={get(tooltips, 'profileNode.forest500Score')}
-                      position="bottom"
-                    />
-                  </span>
-                </div>
-                <div className="value forest-500-score">
-                  {Array.from({ length: 5 }).map((_, index) => (
-                    <Icon
-                      color="grey"
-                      className="circle-icon"
-                      key={`circle${index}`}
-                      icon={`icon-circle-${forest500 > index ? 'filled' : 'empty'}`}
-                    />
-                  ))}
-                </div>
-              </div>
-            )}
-            {typeof zeroDeforestation !== 'undefined' && (
-              <div className="stat-item">
-                <Text transform="uppercase" variant="mono" as="div" className="legend">
-                  ZERO DEFORESTATION COMMITMENT
-                  <span>
-                    <HelpTooltip
-                      text={get(tooltips, 'profileNode.zeroDeforestationCommitment')}
-                      position="bottom"
-                    />
-                  </span>
-                </Text>
-                {zeroDeforestation.toLowerCase() !== 'none' ? (
-                  <div className="value">
-                    <Icon color="grey" icon="icon-check-circle" className="icon-check-circle" />
-                    <Text
-                      transform="uppercase"
-                      variant="mono"
-                      as="span"
-                      weight="bold"
-                      size="lg"
-                      className="stats-text"
-                    >
-                      {zeroDeforestation}
-                    </Text>
-                  </div>
-                ) : (
-                  <div className="value">
-                    <Icon color="grey" icon="icon-no-circle" className="icon-no" />
-                    <Text
-                      transform="uppercase"
-                      variant="mono"
-                      as="span"
-                      weight="bold"
-                      className="stats-text"
-                    >
-                      NONE
-                    </Text>
-                  </div>
-                )}
-              </div>
-            )}
-          </div>
+          {!isEmpty(
```

Comment:
isEmpty could be replaced with a few more lines, besides that, using an expression inside isEmpty is weird.